testsuite/gnat2goto/README.md

```diff
@@ -47,6 +47,26 @@ See also:
 * http://llvm.org/docs/CommandGuide/lit.html
 * http://www.airs.com/blog/archives/499
 
+CBMC submodule
+--------------
+
+The internmediate JSON files generated by gnat2goto are strongly coupled to
```

Comment:
*intern*mediate -> `intermediate`

Reply:
I should use a spell checker.... Fixed, thanks.